tests/test_stimuli.py

```diff
@@ -106,3 +126,128 @@ class TestMarques2020V1Properties:
     def test_num_stimuli(self, identifier, num_stimuli):
         stimulus_set = brainio.get_stimulus_set(identifier)
         assert len(stimulus_set) == num_stimuli
+
+
+@pytest.mark.private_access
+@pytest.mark.slow
+class TestGeirhos2021:
+
+    # test stimulus_set data:
+    @pytest.mark.parametrize('identifier', [
+        'colour',
+        'contrast',
+        'cue-conflict',
+        'edge',
+        'eidolonI',
+        'eidolonII',
+        'eidolonIII',
+        'false-colour',
+        'high-pass',
+        'low-pass',
+        'phase-scrambling',
+        'power-equalisation',
+        'rotation',
+        'silhouette',
+        'stylized',
+        'sketch',
+        'uniform-noise',
+    ])
+    def test_stimulus_set_exist(self, identifier):
+        full_name = f"brendel.Geirhos2021_{identifier}"
+        stimulus_set = brainio.get_stimulus_set(full_name)
+        assert stimulus_set is not None
+        assert stimulus_set.identifier == full_name
+
+    # test the number of images
+    @pytest.mark.parametrize('identifier, num_images', [
+        ('colour', 1280),
+        ('contrast', 1280),
+        ('cue-conflict', 1280),
+        ('edge', 160),
+        ('eidolonI', 1280),
+        ('eidolonII', 1280),
+        ('eidolonIII', 1280),
+        ('false-colour', 1120),
+        ('high-pass', 1280),
+        ('low-pass', 1280),
+        ('phase-scrambling', 1120),
+        ('power-equalisation', 1120),
+        ('rotation', 1280),
+        ('silhouette', 160),
+        ('stylized', 800),
+        ('sketch', 800),
+        ('uniform-noise', 1280),
+    ])
+    def test_num_images(self, identifier, num_images):
+        stimulus_set = brainscore.get_stimulus_set(f"brendel.Geirhos2021_{identifier}")
+        assert len(np.unique(stimulus_set['image_id'].values)) == num_images
+
+    # tests assembly coords for the 14 "normal" sets:
+    @pytest.mark.parametrize('identifier, length', [
+        ('colour', 1280),
+        ('contrast', 1280),
+        ('eidolonI', 1280),
+        ('eidolonII', 1280),
+        ('eidolonIII', 1280),
+        ('false-colour', 1120),
+        ('high-pass', 1280),
+        ('low-pass', 1280),
+        ('phase-scrambling', 1120),
+        ('power-equalisation', 1120),
+        ('rotation', 1280),
+        ('stylized', 800),
+        ('sketch', 800),
+        ('uniform-noise', 1280),
+    ])
```

Comment:
same as above, delete this, get rid of the `length` field and instead test `hasattr(stimulus_set, field)`